src/Extension/EscaperExtension.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-platesrenderer for the canonical source repository
+ * @copyright Copyright (c) 2016-2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-platesrenderer/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Plates\Extension;
+
+use League\Plates\Engine;
+use League\Plates\Extension\ExtensionInterface;
+use Zend\Escaper\Escaper;
+use Zend\Escaper\Exception\InvalidArgumentException;
+
+class EscaperExtension implements ExtensionInterface
+{
+    /**
+     * Register functions with the Plates engine.
+     *
+     * Registers:
+     *
+     * - escapeHtml($string) : string
+     * - escapeHtmlAttr($string) : string
+     * - escapeJs($string) : string
+     * - escapeCss($string) : string
+     * - escapeUrl($string) : string
+     *
+     * @param Engine $engine
+     * @return void
+     * @throws InvalidArgumentException
+     */
+    public function register(Engine $engine)
+    {
+        $escaper = new Escaper();
```

Comment:
We should likely allow providing an encoding to use with the escaper. While the default of UTF-8 is fine, being able to specify another is something we definitely want within a template engine.

That could be done via a constructor parameter.